velox/docs/functions/spark/decimal.rst

```diff
@@ -2,47 +2,92 @@
 Decimal Operators
 =================
 
-When calculating the result precision and scale of arithmetic operators,
-the formulas follow Hive which is based on the SQL standard and MS SQL:
+The result precision and scale computation of arithmetic operators contains two stages.
+In the first stage, the formulas follow Hive which is based on the SQL standard and MS SQL.
+In the second stage, caps p at 38 and reduces scale or not according to the allowing precision loss mode.
+
+Decimal Precision and Scale Computation Formulas
+------------------------------------------------
+
+The SQL standard and MS SQL:
```

Comment:
Do we need to refer to MS SQL? I see it is valuable to match SQL standard, but not sure if there is a lot of value in matching MS SQL and I don't believe we do that anywhere else.

Reply:
I think not, Spark also does not refer to MS Sql, I will drop it.

Reply:
FYI. Below links are from Spark:
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala#L413-L419.